### ISSUE TITLE
implement support for multiple obs docs

### ISSUE THREAD

**Issue by caseyjlaw** [demorest/evla_mcast#2]
(adding this formally after busy week discussion)

The scan configuration is currently defined for a single vci+obs+ant combination. Support for two (or more?) obs docs would allow us to measure start and stop times for a scan. This would make vysmaw commensal applications much easier to write. 

The main outcome may be a new property "stopTime".

**Comment by demorest** [comment]
Ok, I added `stopTime` to `ScanConfig,` and code to `Controller` so that `stopTime` gets filled in when the next Obs doc comes in.  

I was thinking that the simplest way to deal with processing where `stopTime` is needed would be to add an option so that the `Controller.handle_config()` is not called until the stop time is known; this will generally be right at the start of the scan.  Does that sound like it will work to you?

**Comment by caseyjlaw** [comment]
That makes sense to me. How about defining ```is_complete``` to consider whether the start and stop times are known? The controller could then check that before starting ```handle_config```.
Since we're already at the point where "complete" is defined for multiple docs, we might as well define it for xml+vci+obs+obs, which is when a scan is completely defined.

**Comment by demorest** [comment]
Ok, this is implemented and pushed to my branch.  I'm still testing a bit, will close this when I think things are working OK, unless you have any other suggestions.

**Comment by demorest** [comment]
I think this is basically working.  We need to think a little bit about how to handle the special "finish" scans that come at the end of an SB.  I'll open a new issue about this.
